blur-environment.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04 AS base-image
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.8-cudnn8-ubuntu22.04 AS base-image
```

Comment:
was this causing incompatibilities?

Reply:
Yes, because we decided to use Python 3.9. So we needed to also update the image :)